### ISSUE TITLE
[BLOCKED] sohkd: configuration files present duplicated content

### ISSUE THREAD

**Issue by albertotirla** [odilia-app/odilia#80]
so, it seems the configuration files for sohkd contain duplicated data, perhaps the duplicates should be either removed or the rational behind this choice should be explained. For example, the following files contain data present in sohkdrc

* [arrow-granularities.conf](https://github.com/odilia-app/odilia/blob/main/sohkd/arrow-granularities.conf)
* [common-keys.conf](https://github.com/odilia-app/odilia/blob/main/sohkd/common-keys.conf)

if sohkdrc also includes the same information as those two files, do the duplicates have to still exist? if the purpose is to separate key configurations that logically belong in different categories, then why does the monolithic sohkdrc still exist?

this is a low-priority issue, so won't put it in the milestone, unless it can be proven that this is the cause for #79

**Comment by TTWNO** [comment]
The sohkd config parser does not allow imports more than once, or more than one levrl deep.

See: https://github.com/waycrate/swhkd/issues/177

**Comment by albertotirla** [comment]
since we cloned sohkd, I think it should be possible to modify as we wish, as we did when adding json command support, so such limitations shouldn't be a thing anymore. While we may not be able to consider it the original sohkd, it's a thing that happens sometimes with other projects as well, we're stretching the program and adding modifications to it, in such a way that it covers our use case. Practically, the moment we cloned sohkd, we basically made it a new program, with original commits and attribution to the author of course, but that doesn't change the fact. So, I think it could be changed? I'm thinking of the screenreader specifying configuration and sending it over json, so then there's only odilia.conf and some dynamic addon keybindings to wory about, but that might have to change the way sohkd works and some internals. Thing is, I don't think this should be a blocker, opinions?
